pmm-app/src/pmm-dbaas/components/DBCluster/EditDBClusterModal/EditDBClusterModal.types.ts

```diff
@@ -0,0 +1,21 @@
+import { DBCluster } from '../DBCluster.types';
```

Comment:
This file doesn't need to be duplicated. Use the one in the AddDBClusterModal folder instead.

Reply:
Decided to discuss it on a next sync call